README.md

```diff
@@ -121,6 +121,19 @@ $ mongod --dbpath ./data/db --logpath ./data/mongod.log --port 27017 --logappend
 If you encounter `"Too many open files"` errors when running the tests then you will need to increase 
 the number of available file descriptors prior to starting mongod as described in [https://docs.mongodb.com/manual/reference/ulimit/](https://docs.mongodb.com/manual/reference/ulimit/)
 
+## IntelliJ IDEA
+
+A couple of manual configuration steps are required to run the code in IntelliJ:
+
+- Java 9 is required to build and compile the source.
+
+- **Error:** `java: cannot find symbol: class SNIHostName location: package javax.net.ssl`<br>
+ **Fix:** Settings > Build, Execution, Deployment > Compiler > Java Compiler - untick "Use '--release' option for cross-compilation (Java 9 and later)"
+
+- **Error:** `java: package com.mongodb.internal.build does not exist`<br>
```

Comment:
What about recommending to set this task to run `Before Build` by right-clicking and selecting `Execute Before Build`?  That seems less invasive than the second approach and more permanent than the first.

Reply:
Added. I haven't set a preferred way on purpose, as I don't really have one :)